src/clustering/run_clust.jl

```diff
@@ -372,14 +372,14 @@ function run_clust_kmedoids_exact_medoid(
     data_norm::ClustDataMerged,
     n_clust::Int,
     iterations::Int;
-    gurobi_opt=0
+    kmexact_optimizer=0
```

Comment:
Can we not use:
``kmexact_optimizer::DataType=DataType`` or something else defining the Type?

Reply:
Not sure what you are trying to point to. 
The issue we had was that the optimizer object itself does not have a specific type..

Reply:
I think that changed with the new JuMP update (https://github.com/YoungFaithful/CapacityExpansion.jl/blob/d23ffab3f915d6f02e128774b6d2ae82d1c358f0/src/optim_problems/run_opt.jl#L86)

Reply:
That's great. Once we have done the renaming of the package, let's make that change. Opening an issue #111